components/Storyblok/SbHomepageThemeSection/SbHomepageThemeSection.styles.ts

```diff
@@ -2,7 +2,7 @@ import { cnb } from 'cnbuilder';
 
 export const root = 'relative overflow-hidden';
 export const bgImage = 'absolute top-0 left-0 w-full h-full object-cover';
-export const overlay = (hasBgGradient?: boolean) => cnb('absolute top-0 left-0 w-full h-full z-10', hasBgGradient ? 'bg-gradient-to-t' : '');
+export const overlay = (hasBgGradient?: boolean) => cnb('absolute top-0 left-0 w-full h-full z-10', hasBgGradient ? 'bg-gradient-to-b' : '');
```

Comment:
Previously I had the gradient direction going from bottom to top to take advantage of not having to write `to-transparent` TW class (no need to write this if fading to transparent). However, after testing real example pages, I found that we rarely would start with a transparent color at the top anyway so no need for me to do this reverse gradient trick.